src/Chip/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./Chip";
+export * from "./common";
```

Comment:
I'd consider slightly tweaking this structure to make all exported components on the same level, any thoughts?
```
 Chip/
   Chip.tsx
   ChipMovable.tsx
   ChipRemovable.tsx
   ChipSwatch.tsx
   common/
     // all private helpers/components go here
```

Reply:
That's actually a good idea - we don't need to keep up two `index.ts` files + it's clear what is exported and what is not (maybe we should write down a code style rule to do that in the future 😅). I would only use some other name than `common`. I used that because `ChipMovable`, `ChipRemovable`, `ChipSwatch` are "common" variants of the Chip 😉 . Maybe we should use something like `shared` or `private` so that it's clear library consumers shouldn't use those files

Reply:
I don't think that's necessary, if we export everything at top level (`import { x } from "@saleor/macaw-ui` then importing via `import x from "@saleor/macaw-ui/dist/Chip/utils"` looks pretty clear to me that author didn't want consumer to use deep imports. And even if consumer does it anyway, it's, well, their problem.

Reply:
Btw I was referring to `shared` and `private` directories, I'd go for flat structure too.

Reply:
@dominik-zeglen @jwm0